Python gateway_looper_unittest being flaky

```
../../../../../../../../../../heron/common/tests/python/basics/gateway_looper_unittest.py F

=================================== FAILURES ===================================
________________________ GatewayLooperTest.test_wakeup _________________________

self = <basics.gateway_looper_unittest.GatewayLooperTest testMethod=test_wakeup>

    def test_wakeup(self):
      sleep_times = [0.1, 0.3, 0.5, 1.0, 3.0, 5.0]
      for sleep in sleep_times:
        start, end = self.prepare_wakeup_test(sleep)
>       self.assertAlmostEqual(start + sleep, end, delta=0.01)
E       AssertionError: 1487660442.37712 != 1487660442.393256 within 0.01 delta
```

--- a/heron/common/tests/python/basics/gateway_looper_unittest.py
+++ b/heron/common/tests/python/basics/gateway_looper_unittest.py
@@ -34,7 +34,7 @@ class GatewayLooperTest(unittest.TestCase):
     sleep_times = [0.1, 0.3, 0.5, 1.0, 3.0, 5.0]
     for sleep in sleep_times:
       start, end = self.prepare_wakeup_test(sleep)
-      self.assertAlmostEqual(start + sleep, end, delta=0.01)
+      self.assertAlmostEqual(start + sleep, end, delta=0.02)
 
   def prepare_wakeup_test(self, sleep, poll_timeout=30.0):
     looper = GatewayLooper(socket_map={})

